test/soca/socaincr2mom6.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+srcdir=$1
+
+data="/home/gvernier/wcda-sprint/wrk"
```

Comment:
brilliant work once again @guillaumevernieres

Reply:
We should strive for portability, after all